extensions/amp-analytics/0.1/linker.js

```diff
@@ -98,6 +169,21 @@ function serialize(pairs) {
       .join(DELIMITER);
 }
 
+/**
+ * Deserialize the serializedIds and return keyValue pairs.
+ * @param {string} serializedIds
+ * @return {!Object<string, string>}
+ */
+function deserialize(serializedIds) {
+  const keyValuePairs = {};
+  const params = serializedIds.split(DELIMITER);
+  for (let i = 0; i < params.length; i += 2) {
+    const key = params[i];
```

Comment:
key has to be base64 charset. see `KEY_VALIDATOR`

Reply:
check added.